src/main/gui/main-window.ts

```diff
@@ -478,12 +498,13 @@ export const createMainWindow = async (args: OpenArguments, settings: ChainnerSe
         roundedCorners: true,
         webPreferences: {
             webSecurity: false,
-            nodeIntegration: true,
+            nodeIntegration: false,
             nodeIntegrationInWorker: true,
-            contextIsolation: false,
+            contextIsolation: true,
+            preload: path.join(__dirname, '/preload.js'),
         },
         icon: `${__dirname}/../public/icons/cross_platform/icon`,
-        show: false,
+        show: true,
```

Comment:
Why did you change this?

Reply:
For debugging purposes when j first started this, and then I forgot to change it back. Oops